posthog/settings/data_stores.py

```diff
@@ -0,0 +1,143 @@
+import os
```

Comment:
Nice work! Can we split this even further? e.g. postgresql.py/clickhouse.py....

Reply:
For sure - I only split to the point where it wasn't painful for me anymore, if we want to/there's need we can refactor further. However I'd keep that in separate PRs to keep this as much copy-pasta code as possible.